web/src/lib/components/shared-components/settings/setting-switch.svelte

```diff
@@ -30,9 +33,8 @@
       {/if}
     </div>
 
-    <p class="text-sm dark:text-immich-dark-fg">{subtitle}</p>
-    <slot />
+    <label class="text-sm dark:text-immich-dark-fg" for={sliderId}>{subtitle}</label>
```

Comment:
This shouldn't be a label. Using aria-describedby or something alike would be more appropriate

Reply:
Yeah... so I was debating this too. Using `aria-describedby` makes sense to me, but many of the settings sliders are really using the subtitle as the label of the slider. How would you feel about a follow-up PR to improve the copy on some of the sliders to make the titles better labels, and only include a subtitle if it's needed to provide additional context?

![slider-logging](https://github.com/immich-app/immich/assets/45583362/e39cf405-7112-4948-b498-8e7eb4163724)
![slider-scanning](https://github.com/immich-app/immich/assets/45583362/69fc2399-09e2-4cb8-b173-7aa93d641eec)
![slider-sharing](https://github.com/immich-app/immich/assets/45583362/d56db730-bf3f-4dc9-803c-abd3d192bb3e)
![slider-storage-template](https://github.com/immich-app/immich/assets/45583362/42c99038-7ec6-4b3b-b4e0-60529c1c56b0)

Reply:
Ah, I understand now why it was changed to a label. The contents of `aria-describedby` should get announced along with the label, but I'm also fine doing this in a separate PR

Reply:
I've updated the PR to use `aria-describedby` instead. PR soon to start a discussion on label copy!